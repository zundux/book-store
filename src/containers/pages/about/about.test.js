import React from "react";
import About from "./about";

describe("About page", () => {

  it("About page", () => {
    const component = shallow(<About/>);
    matchSnapshot(component);
    // todo check page header
    // todo check page title
  });
});