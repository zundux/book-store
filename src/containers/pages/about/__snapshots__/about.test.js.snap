// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page About page 1`] = `
<div>
  <h1>
    About Us
  </h1>
  <p>
    Created for demo purpose
  </p>
</div>
`;
